gsuite/resource_group_members.go

```diff
@@ -0,0 +1,278 @@
+package gsuite
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	directory "google.golang.org/api/admin/directory/v1"
+)
+
+func resourceGroupMembers() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGroupMembersCreate,
+		Read:   resourceGroupMembersRead,
+		Update: resourceGroupMembersUpdate,
+		Delete: resourceGroupMembersDelete,
+
+		Schema: map[string]*schema.Schema{
+			"group": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"member": {
+				Type:     schema.TypeSet,
+				Required: true,
+				Elem: &schema.Resource{
+					Schema: schemaMember,
+				},
+			},
+		},
+	}
+}
+
+func resourceGroupMembersRead(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	members, err := getApiMembers(d.Id(), config)
+
+	if err != nil {
+		return err
+	}
+
+	d.Set("group", d.Id())
+	d.Set("member", membersToCfg(members))
+	return nil
+}
+
+func resourceGroupMembersCreate(d *schema.ResourceData, meta interface{}) error {
+	log.Printf("[DEBUG]: Creating gsuite_group_members")
+	gid, err := createOrUpdateGroupMembers(d, meta)
+
+	if err != nil {
+		return err
+	}
+
+	d.SetId(gid)
+	return resourceGroupMembersRead(d, meta)
+}
+
+func resourceGroupMembersUpdate(d *schema.ResourceData, meta interface{}) error {
+	log.Printf("[DEBUG]: Updating gsuite_group_members")
+	_, err := createOrUpdateGroupMembers(d, meta)
+
+	if err != nil {
+		return err
+	}
+	return resourceGroupMembersRead(d, meta)
+}
+
+func resourceGroupMembersDelete(d *schema.ResourceData, meta interface{}) error {
+	log.Printf("[DEBUG]: Deleting gsuite_group_members")
+	config := meta.(*Config)
+
+	for _, raw_member := range d.Get("member").(*schema.Set).List() {
+		member := raw_member.(map[string]interface{})
+		deleteMember(member["email"].(string), d.Id(), config)
+	}
+
+	d.SetId("")
+	return nil
+}
+
+func membersToCfg(members *directory.Members) []map[string]interface{} {
```

Comment:
```
func membersToCfg(members []*directory.Member) []map[string]interface{} {
	if members == nil {
		return nil
	}

	finalMembers := make([]map[string]interface{}, 0, len(members))

	for _, m := range members {
		finalMembers = append(finalMembers, map[string]interface{}{
			"email":  m.Email,
			"etag":   m.Etag,
			"kind":   m.Kind,
			"status": m.Status,
			"type":   m.Type,
			"role":   m.Role,
		})
	}

	return finalMembers
}
```
I think that's how this should look like?

Reply:
After applying this fix I can `make` and install, and it works successfully :+1:

Reply:
Yes, you are right. I made changes without trying to compile it again :(